### ISSUE TITLE
Turn landmarks into rectangles and make them stop moving

### ISSUE THREAD

**Issue by mariojerez** [mariojerez/RL-MultiAgent-PettingZoo#2]
Turn landmarks into non-moving rectangles, creating tunnel.

**Comment by mariojerez** [comment]
Turned landmarks into rectangles by editing SimpleEnv class in simple_env.py. Before, it set all entities to circles, regardless of whether or not they were agents or Landmarks. Added the below code to differentiate them:

```python
if isinstance(entity, Agent):
   pygame.draw.circle(self.screen, entity.color * 200, (x, y), radius)
   pygame.draw.circle(self.screen, (0, 0, 0), (x, y), radius, 1)  # borders
elif isinstance(entity, Landmark):
   pygame.draw.rect(self.screen, entity.color * 200, (x, y, radius * 10, radius * 20))
   pygame.draw.rect(self.screen, (0, 0, 0), (x, y, radius, radius * 10), 20)
```

**Comment by mariojerez** [comment]
See description in issue #5

**Comment by mariojerez** [comment]
Hey @aryanj2003.

The landmarks aren't actually moving.

If you add this code to RL_switch_sides.py, you'll see from the print statements that their positions don't actually change.
```python3
while env.agents:
    # this is where you would insert your policy
    actions = {agent: env.action_space(agent).sample() for agent in env.agents}

    observations, rewards, terminations, truncations, infos = env.step(actions)
    landmark1 = env.aec_env.world.landmarks[0]
    landmark2 = env.aec_env.world.landmarks[1]
    print(f"{landmark1.name} position: {landmark1.state.p_pos}, velocity: {landmark1.state.p_vel}; {landmark2.name} position: {landmark2.state.p_pos}, velocity: {landmark2.state.p_vel}")
env.close()
```

It looks like they are moving because the draw() method of the SimpleEnv class in simple_env.py tries to rescale the scene by keeping the agents centered and zooming in and out as they approach the boundaries. I think we should get rid of this feature because it's just confusing, and we should probably create hard boundaries in our environment, rather than having it go from negative infinity to positive infinity. These other things will be separate Issues, so for now, you can just stop it from rescaling the view. Here's some rough code that does the job. I just commented out the parts that did the scaling.

```python3
def draw(self):
        # clear screen
        self.screen.fill((255, 255, 255))

        # update bounds to center around agent
        #all_poses = [entity.state.p_pos for entity in self.world.entities]
        #cam_range = np.max(np.abs(np.array(all_poses)))

        #TODO: Get rid of scaling
        #TODO: Adjust environment so that it is bounded between -1 and 1

        # The scaling factor is used for dynamic rescaling of the rendering - a.k.a Zoom In/Zoom Out effect
        # The 0.9 is a factor to keep the entities from appearing "too" out-of-bounds
        #scaling_factor = 0.9 * self.original_cam_range / cam_range

        # update geometry and text positions
        text_line = 0
        for e, entity in enumerate(self.world.entities):
            # geometry
            x, y = entity.state.p_pos
            y *= (
                -1
            )  # this makes the display mimic the old pyglet setup (ie. flips image)
            x = (
                (x / self.original_cam_range) * self.width // 2 * 0.9
            )  # the .9 is just to keep entities from appearing "too" out-of-bounds
            y = (y / self.original_cam_range) * self.height // 2 * 0.9
            x += self.width // 2
            y += self.height // 2

            # 350 is an arbitrary scale factor to get pygame to render similar sizes as pyglet
            if self.dynamic_rescaling:
                radius = entity.size * 350 #* scaling_factor
            else:
                radius = entity.size * 350
            
            if isinstance(entity, Agent):
                pygame.draw.circle(self.screen, entity.color * 200, (x, y), radius)
                pygame.draw.circle(self.screen, (0, 0, 0), (x, y), radius, 1)  # borders
            elif isinstance(entity, Landmark):
                pygame.draw.rect(self.screen, entity.color * 200, (x, y, radius * 10, radius * 20))
                pygame.draw.rect(self.screen, (0, 0, 0), (x, y, radius, radius), 1)

            #TODO: Add this assert back when we limit agents from leaving boundary
            #assert (
            #    0 < x < self.width and 0 < y < self.height
            #), f"Coordinates {(x, y)} are out of bounds."

            # text
            if isinstance(entity, Agent):
                if entity.silent:
                    continue
                if np.all(entity.state.c == 0):
                    word = "_"
                elif self.continuous_actions:
                    word = (
                        "[" + ",".join([f"{comm:.2f}" for comm in entity.state.c]) + "]"
                    )
                else:
                    word = alphabet[np.argmax(entity.state.c)]

                message = entity.name + " sends " + word + "   "
                message_x_pos = self.width * 0.05
                message_y_pos = self.height * 0.95 - (self.height * 0.05 * text_line)
                self.game_font.render_to(
                    self.screen, (message_x_pos, message_y_pos), message, (0, 0, 0)
                )
                text_line += 1
```

**Comment by mariojerez** [comment]
Resolved with commit Commit b878b76.

@aryanj2003 in the future, please describe the changes you're making in the commit message. Reference the issue number by putting a hashtag before the issue number, like "#2". Also describe changes in more detail in your pull requests.
